### PR TITLE
Add Characteristic Cycling Speed Cadence

### PR DESCRIPTION
**Pull request by joeblau**
Add Characteristic Cycling Speed Cadence

### CODE REVIEW

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 1]
```diff
@@ -23,61 +23,71 @@ final public class CharacteristicCyclingSpeedCadence: Characteristic {
...
     
-    
-    public enum Flags: UInt8 {
-        case wheelPresent = 0
-        case crankPresent = 1
-    }
-    
     private(set) public var flags: Flags
+
+    private(set) public var cumulativeWheelRevoluations: UInt32
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift]: These can be optional as you have the Flag to tell you what is available

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 2]
```diff
@@ -23,61 +23,71 @@ final public class CharacteristicCyclingSpeedCadence: Characteristic {
...
     /// Characteristic UUID String
     public var uuidString: String { Self.uuidString }
     
-    
-    public enum Flags: UInt8 {
-        case wheelPresent = 0
-        case crankPresent = 1
-    }
-    
     private(set) public var flags: Flags
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift:49]: All the Vars need Comments to explain what they are..
This is a Framework for others so be kind and provide information

**Review by khoogheem — COMMENTED** [review]
Also needs to update the `.jazzy` file and run jazzy to include the documents

**Comment by joeblau** [comment]
> Also needs to update the `.jazzy` file and run jazzy to include the documents

When I run Jazzy, I get this error
```
Using config file /Users/joeblau/Developer/ridevelo/src/bluetooth-message-protocol/.jazzy.yaml
Running swift build
Build failed, saved `swift build` log file: /var/folders/8q/qq0pfsnx13l90x4zfchl5nrm0000gn/T/swift-build-A5B15FF3-DC76-49A8-B782-A143A3856EDE.log
Failed to generate documentation
Traceback (most recent call last):
	7: from /usr/local/bin/jazzy:23:in `<main>'
	6: from /usr/local/bin/jazzy:23:in `load'
	5: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.6/bin/jazzy:15:in `<top (required)>'
	4: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.6/lib/jazzy/doc_builder.rb:79:in `build'
	3: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.6/lib/jazzy/doc_builder.rb:79:in `chdir'
	2: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.6/lib/jazzy/doc_builder.rb:81:in `block in build'
	1: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.6/lib/jazzy/sourcekitten.rb:266:in `run_sourcekitten'
/Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.6/lib/jazzy/executable.rb:36:in `execute_command': /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.6/bin/sourcekitten ["doc", "--spm", "--module-name", "BluetoothMessageProtocol", "--", "-scheme", "BluetoothMessageProtocol-Package"] (RuntimeError)

Running swift build

Build failed, saved `swift build` log file: /var/folders/8q/qq0pfsnx13l90x4zfchl5nrm0000gn/T/swift-build-A5B15FF3-DC76-49A8-B782-A143A3856EDE.log

Failed to generate documentation
```

Then from the file
```
error: The value 'BluetoothMessageProtocol-Package' is invalid for '-c <configuration>'
Usage: swift build <options>
  See 'build --help' for more information.
```

I haven't used jazzy before so I'm not sure what it's complaining about.

**Comment by khoogheem** [comment]
> > Also needs to update the `.jazzy` file and run jazzy to include the documents
> 
> When I run Jazzy, I get this error
> 
hmm... try doing a swift package generate-xcodeproj .. it might be tied to the Xcode project...

**Comment by joeblau** [comment]
That was it!

**Comment by joeblau** [comment]
The version of Jazzy I ran changed every single file 😅. I would rather submit this code change independently and then run Jazzy to update all of the docs in a separate PR. There is also a spelling error I want to fix.

**Comment by khoogheem** [comment]
still need to document the vars in the Characteristic.. 
I typically would like the docs updates at the same time...

**Comment by khoogheem** [comment]
I will look at it later this week.. have "real life" to deal with

**Comment by joeblau** [comment]
No problem and no rush. Take your time and thanks for your feedback.

**Pull request merged**